.azure-pipelines/cpu-unit-test.yml

```diff
@@ -7,10 +7,12 @@ trigger:
 
 strategy:
   matrix:
-    python-3.6:
-      imageTag: '3.6'
+    python-3.8:
+      imageTag: '3.8'
     python-3.7:
       imageTag: '3.7'
+    python-3.6:
+      imageTag: '3.6'
     # TODO
     #python-latest:
```

Comment:
keep ascending order? add 3.8 in the end

Reply:
updated